Cargo.toml

```diff
@@ -5,22 +5,27 @@ authors = ["Victor Berger <victor.berger@thalesgroup.com>"]
 license = "MIT"
 
 [dependencies]
-wayland-server = "0.9.1"
+wayland-server = "0.9.4"
 nix = "0.7.0"
 xkbcommon = "0.2.1"
 tempfile = "2.1.5"
 slog = { version = "2.0.0" }
 slog-stdlog = "2.0.0-0.2"
-glutin = { version = "~0.7.4", optional = true }
+libloading = "0.4.0"
+wayland-client = { version = "~0.8.6", optional = true }
+winit = { version = "~0.6.4", optional = true }
 glium = { version = "~0.16.0", optional = true }
 input = { version = "~0.1.1", optional = true }
 clippy = { version = "*", optional = true }
 
+[build-dependencies]
+gl_generator = "0.5"
+
 [dev-dependencies]
 slog-term = "~1.5"
 
 [features]
-default = ["backend_glutin", "backend_libinput", "renderer_glium"]
-backend_glutin = ["glutin", "wayland-server/dlopen"]
+default = ["backend_winit", "backend_libinput", "renderer_glium"]
+backend_winit = ["winit", "wayland-server/dlopen", "wayland-client", "wayland-client/dlopen"]
```

Comment:
I don't think we need both `wayland-client` and `wayland-client/dlopen` here, do we?